### DISCUSSION TITLE
Extension of character limitation via Web UI/UX

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#18547] [General]
From the point of view of the Mastodon project, what is the objection to enabling administrative settings with regard to increasing (or decreasing) the maximum number of characters per post via the web interface as part of a new release version?

**Comment by nightpool** [comment]
how would this be a "simplification"? 

Mastodon's UI and UX is designed with a certain character limit in mind. Larger character limits (and formatted text, long form posts, etc) are not a goal of the software

**Reply by linux-lukas** [reply]
Well, I think it would be useful if the Mastodon project would give the instance admins the option to set the character limit up or down, for example. 

And enable this possibility so that not much has to be changed in the code itself, but this can simply be realised via the Admin Web UI/UX.

**Reply by linux-lukas** [reply]
I realise that this would mean a lot of code restructuring, but I believe that the advantages of this outweigh the disadvantages and it could help the Mastodon project overall to become more attractive, to the extent that public bodies consider running their own instance more often.

**Comment by ClearlyClaire** [comment]
If I understand correctly, what you are asking is an admin option to change the max length of a post. If so, the discussion's title is pretty misleading. Some forks offer such a feature, and it has been proposed in Mastodon (e.g. https://github.com/mastodon/mastodon/issues/12265) but so far, Gargron has declined to have such an option.

**Reply by linux-lukas** [reply]
Yes, that's basically it. I apologise for the misleading title - I will clarify it. 

There are of course good reasons for and against and perhaps a compromise could be reached that is helpful for Gragon and people from the community alike.
